CMakeLists.txt

```diff
@@ -123,12 +123,16 @@ set(CONFIG_H ${CONFIG_H_DIR}/config.h)
 
 # This is intended to make life easier for developers who are working on one
 # specific feature.
-option(DEFAULT_FEATURE_STATE "Specify the default state for most optional features" True)
+option(DEFAULT_FEATURE_STATE "Specify the default state for most optional features" False)
```

Comment:
This should not be changed.

Reply:
JFYI, this is _probably_ the root cause for the failing build checks.

Reply:
I am sorry for this, this was a mistake that happening due some local tests. :(